pkg/commands/loading_branches.go

```diff
@@ -92,15 +92,26 @@ func (b *BranchListBuilder) obtainBranches() []*models.Branch {
 			branch.Pullables = "0"
 		}
 
+		if b.GitCommand.GithubRecentPRs != nil {
+			remoteAndName := strings.SplitN(upstreamName, "/", 2)
+			if len(remoteAndName) == 2 {
+				pr, hasPr := b.GitCommand.GithubRecentPRs[remoteAndName[1]]
```

Comment:
Note to self,
This causes unexpected behaviour when 2 people use the same branch and the other has created a pr with that nam like master.

Reply:
Fixed